packages/plugin-packages/mashroom-portal-sandbox-app/src/js/components/MessageBusSendForm.tsx

```diff
@@ -54,7 +58,11 @@ export default class MessageBusSendForm extends PureComponent<Props> {
     onSubmit(values: FormData, context: FormContext): void {
         const { messageBus, addMessagePublishedBySandbox } = this.props;
         const { topic, message } = values;
-        const data = JSON.parse(message);
+
+        const inputFieldValue : string = (document.querySelector('#mashroom-sandbox-publish-message-message') as HTMLInputElement)?.value;
```

Comment:
Thats kinda weird here - please use a TextareaField instead of a plain input because it can properly participate in the form